src/common/utility.cpp

```diff
@@ -555,22 +557,22 @@ QString Utility::makeConflictFileName(
 {
     QString conflictFileName(fn);
     // Add conflict tag before the extension.
-    int dotLocation = conflictFileName.lastIndexOf('.');
+    int dotLocation = conflictFileName.lastIndexOf(QLatin1Char('.'));
     // If no extension, add it at the end  (take care of cases like foo/.hidden or foo.bar/file)
-    if (dotLocation <= conflictFileName.lastIndexOf('/') + 1) {
+    if (dotLocation <= conflictFileName.lastIndexOf(QLatin1Char('/')) + 1) {
         dotLocation = conflictFileName.size();
     }
 
     QString conflictMarker = QStringLiteral(" (conflicted copy ");
     if (!user.isEmpty()) {
         // Don't allow parens in the user name, to ensure
         // we can find the beginning and end of the conflict tag.
-        const auto userName = sanitizeForFileName(user).replace('(', '_').replace(')', '_');
+        const auto userName = sanitizeForFileName(user).replace(QRegularExpression(QStringLiteral("\\(|\\)")), QStringLiteral("_"));
```

Comment:
Not sure this is more readable ...

if this is supposed to be a performance improvement, have you benchmarked it?